server/app/services/applicant/question/Scalar.java

```diff
@@ -18,14 +18,17 @@
  */
 public enum Scalar {
   CITY("city", ScalarType.STRING),
+  CORRECTED("corrected", ScalarType.STRING),
```

Comment:
**suggestion (blocking):** this looks like it should be a `ScalarType.BOOLEAN`

Reply:
@seanmalbert Are there any other states you think this could be? Otherwise I'll switch this to a bool.

Reply:
@gwendolyngoetz and @bion, 
It may be useful to capture a failed state like we do with service area validation, so CF would know if address correction was attempted but the external service returned a non 200 status. If this fails, we cannot perform service area validation for an address.